monitor.py

```diff
@@ -1337,24 +1356,26 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
 
             elif event_type == 'materials':
                 for category in ('Raw', 'Manufactured', 'Encoded'):
+                    category = cast(Literal['Raw', 'Manufactured', 'Encoded'], category)
```

Comment:
Any reason this one isn't TYPE_CHECKING gated, but one above is ?

Reply:
Because who needs consistency! fixed by removing the guard on the other one (as there were more without than with)